distribution/src/main/resources/config/jvm.options

```diff
@@ -84,13 +81,16 @@
 # ensure the directory exists and has sufficient space
 ${heap.dump.path}
 
-## GC logging
+## JDK 8 GC logging
+
+8:-XX:+PrintGCDetails
+8:-XX:+PrintGCDateStamps
+8:-XX:+PrintTenuringDistribution
+8:-XX:+PrintGCApplicationStoppedTime
+8:-Xloggc:${loggc}
+8:-XX:+UseGCLogFileRotation
+8:-XX:NumberOfGCLogFiles=32
+8:-XX:GCLogFileSize=64m
 
--XX:+PrintGCDetails
--XX:+PrintGCDateStamps
--XX:+PrintTenuringDistribution
--XX:+PrintGCApplicationStoppedTime
--Xloggc:${loggc}
--XX:+UseGCLogFileRotation
--XX:NumberOfGCLogFiles=32
--XX:GCLogFileSize=64m
+# JDK 9+ GC logging
+9-:-Xlog:age*=trace,gc*,safepoint:file=${loggc}:utctime,pid,tags:filecount=32,filesize=64m
```

Comment:
I tried this config on Oracle JDK 9.0.1 and I could never get any log output for the tenuring distribution. If you change this line to:

```
+9-:-Xlog:gc*,gc+age=trace,safepoint:file=${loggc}:utctime,pid,tags:filecount=32,filesize=64m
```

(alternatively `-Xlog:gc*,age*=trace...` works too, I just opted here for the more specific `gc+age`)

Reply:
Thanks @danielmitterdorfer, I pushed d548403ad205b116d29a639187c29b235ed5eef2.

Reply:
Thanks!